libcontainer/cgroups/fs/blkio.go

```diff
@@ -226,12 +230,15 @@ func getStats(path string, stats *cgroups.Stats) error {
 	if blkioStats, err = getBlkioStat(filepath.Join(path, "blkio.throttle.io_service_bytes")); err != nil {
 		return err
 	}
-	stats.BlkioStats.IoServiceBytesRecursive = blkioStats
+	//	stats.BlkioStats.IoServiceBytesRecursive = blkioStats
```

Comment:
After a second thought, I think we might reuse `IoServiceBytesRecursive` for some reasons, it's been there since the old libcontainer days, maybe upper level monitor tools don't have to care if bandwidth or throttle is used by blkio. And your change will break these tools.